libs/community/tests/unit_tests/vectorstores/test_sqlserver.py

```diff
@@ -146,3 +146,84 @@ def test_that_a_document_entry_without_metadata_will_be_added_to_vectorstore(
     ]
     result = store.add_documents(docs)
     assert len(result) == len(docs)
+
+
+def test_sqlserver_delete_text_by_id_valid_ids_provided(
+    store: SQLServer_VectorStore,
+) -> None:
+    """Test that delete API deletes texts by id."""
+    texts = [
+        "Good review",
+        "new books",
+        "table",
+        "Sunglasses are a form of protective eyewear.",
+        "It's a new year.",
+    ]
+
+    # List of ids is 3 and is less than len(texts) which is 5.
```

Comment:
I don't think this comment applies here.

Reply:
Fixed